### DISCUSSION TITLE
Ideapad Flex 5-14IIL05 - 0.8.4

### DISCUSSION THREAD

**Discussion by RobyRew** [RobyRew/Lenovo-Ideapad-Flex-5-14IIL05_Hackintosh_OpenCore#9] [Announcements]
### News / Fixes:
- **Removed** all @andrehessels work and builded myself from scratch.
- Booted **macOS Ventura Beta 7** with any problems
- Used the same USB Map (but needs a fix in the future)
- Detected iGPU acceleration issues, I need another iCL Framebuffer ([what im talking about](https://github.com/acidanthera/WhateverGreen/blob/master/Manual/FAQ.IntelHD.en.md#intel-iris-plus-graphics-ice-lake-processors))
- Internal SSD is working.
- WiFi and Bluetooth are now "natively" working
- iCloud, iMessages, FaceTime, etc.. are working (but the iCloud must be signed in after installation on Settings)
- config.plist well done.
- Fixed flickering issues on login

### Not Working:
- Sleep is currently not working and it will reboot if you press the button from Apple Logo.
- Trackpad
- Internal Microphone (Im sure needs a different layout for AppleALC)
- Touch Display
- Finger Unlock (Never would)

---

### Help is needed for a trackpad & Display Touch fix and more improvements.

---

### Other Issues:
- Ive noticed some TSC issues on latest boot stage might need a fix in the future
- Also the BIOS is crazy poor.

With this EFI you should be able to boot into macOS Big Sur UP to VENTURA BETA 7 and install it with no problem.

Also remember to generate your own SMBIOS.

<hr /><em>This discussion was created from the release <a href='https://github.com/RobyRew/Lenovo-Ideapad-Flex-5-14IIL05_Hackintosh_OpenCore/releases/tag/0.8.4'>Ideapad Flex 5-14IIL05 - 0.8.4</a>.</em>

**Comment by chackoabr** [comment]
Hey Roby, spent hours on getting the usb to boot for some reason, but now im stuck on this. tried monterrey and big sur. Can you help me out?

i have the same laptop as required except i3 10th gen

**Reply by RobyRew** [reply]
sure i can help you, where are you stuck?
can you send a video or screenshots?
cheers!
